src/main/java/com/itrex/java/lab/entity/dto/OfferReportDTO.java

```diff
@@ -0,0 +1,18 @@
+package com.itrex.java.lab.entity.dto;
+
+public interface OfferReportDTO {
+
+    Integer getContractId();
+
+    String getContractDescription();
+
+    Integer getStartPrice();
+
+    Integer getOfferOwnerId();
+
+    String getOfferOwnerName();
+
+    Integer getOfferId();
+
+    Integer getMinOfferPrice();
+}
```

Comment:
I think that in this case the interface does not make sense because you use it to return the fields. The interface is only used if you create a contract for a specific behavior, not for field return. But I may be wrong

Reply:
It is the feature of customizing the result of JPA queries with Spring Data Projection. You can find more information here: [https://www.baeldung.com/jpa-queries-custom-result-with-aggregation-functions#solution_spring_data](url)

Reply:
ERROR: Page not found